multigeiger/multigeiger.ino

```diff
@@ -201,59 +202,59 @@ const unsigned long GMC_dead_time = 190;  // Dead Time of the Geiger Counter. Ha
-         unsigned long last_count_timestamp_2send = millis();
-         unsigned long accumulated_time       = 0;
-         unsigned int  last_GMC_counts        = 0;
-         unsigned int  hvpulsecnt2send        = 0;
-         float         Count_Rate             = 0.0;
-         float         Dose_Rate              = 0.0;
-         float         accumulated_Count_Rate = 0.0;
-         float         accumulated_Dose_Rate  = 0.0;
-         unsigned long lastMinuteLog          = millis();
-         unsigned int  lastMinuteLogCounts    = 0;
-         unsigned int  current_cpm            = 0;
-
-         unsigned long toSendTime             = millis();
-         unsigned long afterStartTime         = 0;
-         unsigned long time2hvpulse           = millis();
-         unsigned long time2display           = millis();
-
-         bool          showDisplay            = SHOW_DISPLAY;
-         bool          speakerTick            = SPEAKER_TICK;
-         bool          ledTick                = LED_TICK;
-         bool          playSound              = PLAY_SOUND;
-         bool          displayIsClear         = false;
-         char          ssid[IOTWEBCONF_WORD_LEN];  // LEN == 33 (2020-01-13)
-         int           haveBME280             = 0;
-         float         bme_temperature        = 0.0;
-         float         bme_humidity           = 0.0;
-         float         bme_pressure           = 0.0;
-         float         GMC_factor_uSvph       = 0.0;
-         portMUX_TYPE  mux_cap_full = portMUX_INITIALIZER_UNLOCKED;
-         portMUX_TYPE  mux_GMC_count = portMUX_INITIALIZER_UNLOCKED;
-         const char*         Serial_Logging_Header = "GEIGER: %10s %15s %10s %9s %9s %8s %9s %9s %9s\r\n";
-         const char*         Serial_Logging_Body   = "GEIGER: %10d %15d %10f %9f %9d %8d %9d %9f %9f\r\n";
-         const char*         Serial_One_Minute_Log_Header = "GEIGER: %4s %10s %29s\r\n";
-         const char*         Serial_One_Minute_Log_Body   = "GEIGER: %4d %10d %29d\r\n";
-         const char*         Serial_Logging_Name   = "GEIGER: Simple Multi-Geiger, Version ";
-         char          revString[25];
-         unsigned int  lora_software_version; 
-         const String        dashes                  = "GEIGER: -------------------------------------------------------------------------------------------------";
-         int           Serial_Print_Mode       = SERIAL_DEBUG;
+volatile unsigned int  isr_GMC_counts_2send = 0;
+volatile unsigned long isr_count_timestamp_2send = micros();
+
+unsigned int  GMC_counts = 0;
+unsigned int  GMC_counts_2send = 0;
+unsigned int  accumulated_GMC_counts = 0;
+unsigned long count_timestamp = millis();
+unsigned long count_timestamp_2send = millis();
+unsigned long last_count_timestamp = millis();
+unsigned long last_count_timestamp_2send = millis();
+unsigned long accumulated_time = 0;
+unsigned int  last_GMC_counts = 0;
+unsigned int  hvpulsecnt2send = 0;
+float         Count_Rate = 0.0;
```

Comment:
ok, my fault, sorry.  Not at the '=', but after the data type (float). That's what  I meant.

Reply:
ok, I'll fix that.